easybuild/easyconfigs/l/libcint/libcint-5.4.0-gfbf-2022b.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'libcint'
+version = '5.4.0'
```

Comment:
There's a 5.5.0 release, as well as 6.1.1, can/should we use that instead?

Reply:
Ah i found why I did it because there is already a libcint in a newer toolchain that has v5.4.0 so I did not know if I could add a newer version to older toolchain. https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/l/libcint

Reply:
Already have a version on a newer toolchain shouldn't block us from picking up releases with bug fixes

5.5.0 has a memleak fix, see https://github.com/sunqm/libcint/releases/tag/v5.5.0, so I would bump to that